tests/specs/install/move_after_install/__test__.jsonc

```diff
@@ -15,8 +15,10 @@
       "output": "5\n"
     },
     {
-      "commandName": "mv",
```

Comment:
No `mv` command on Windows (the Windows CI adds it, but it doesn't exist normally)